README.md

```diff
@@ -152,4 +152,17 @@ Starting a web ui at `localhost:8080` to visualize/analyze the profiling data:
 go tool pprof -http=:8080 /path/to/profiling.out
 ```
 For extra info:
-[link](https://jvns.ca/blog/2017/09/24/profiling-go-with-pprof/)
\ No newline at end of file
+[link](https://jvns.ca/blog/2017/09/24/profiling-go-with-pprof/)
+
+
+### Creating a changelog
+At `./cmd/changelog/main.go` there is a script that can generate a changelog for you.
+
+#### Prerequisites
+It uses both the local git and GitHub`s API to generate the file so:
+- To get info from GitHub you will need to set the `GITHUB_TOKEN` envvar to a GitHub access-token.
```

Comment:
I'm a bit conflicted about using the GitHub API, because the only thing we get is the categories defined in the PRs description. That feels a bit of an overkill.

Reply:
I quite like the categories.